cpp-client/deephaven/client/include/public/deephaven/client/utility/utility.h

```diff
@@ -149,6 +149,20 @@ internal::SeparatedListAdaptor<Iterator, Callback> separatedList(Iterator begin,
  */
 #define DEEPHAVEN_EXPR_MSG(EXPR) (EXPR), #EXPR "@" __FILE__ ":" DEEPHAVEN_STRINGIFY(__LINE__)
 
+/**
+ * If result's status is OK, do nothing. Otherwise throw a runtime error with an informative message.
+ * @param result an arrow::Result
+ * @param optionalMessage An optional message to be included in the exception message.
+ */
+template<typename T>
+void okOrThrow(const arrow::Result<T> &result, const char *message) {
+  if (result.ok()) {
```

Comment:
I think this method can save a little template code bloat and just be:

```
template<typename T>
void okOrThrow(const arrow::Result<T> &result, const char *message) {
  okOrThrow(result.status(), message);
}
```

Reply:
Not sure why I am seeing comments twice... see above.